app/services/user_password_expirer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
thats surprising names for something that creates stuff

Reply:
I get that we are going to take the last password but still, that's not expiring a password by itself.

Reply:
Are you referring to the class name or method name here?

Reply:
I got confused at first by the method name as well here, because `expire_user_password()` is closely followed by `UserPassword.create!()`. 

On second read, though, this makes sense. This line: 

```
password_expired_at: Time.zone.now,
```

is what expires the password. And longterm the plan is to use this table for all passwords (active and expired).